src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -47,9 +47,9 @@
     private $idReaders = array();
 
     /**
-     * @var DoctrineChoiceLoader[]
+     * @var ChoiceListFactoryInterface
      */
-    private $choiceLoaders = array();
+    private $choiceListFactory;
```

Comment:
I suggest keeping the properties in the same order than previously, to avoid messing with ``git blame``